docs/changelog.rst

```diff
@@ -2,6 +2,10 @@
 Changelog
 *********
 
+1.30.1 (2024-06-21)
+-------------------
+- Add stage-packages to ignore for core24
```

Comment:
Maybe something like "Fix list of ignored packages in core24 bases when fetching stage-packages"